src/node_options.cc

```diff
@@ -391,6 +391,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::experimental_global_customevent,
             kAllowedInEnvvar,
             true);
+  AddOption("--experimental-global-navigator",
+            "expose experimental Navigator API on the global scope",
```

Comment:
What's the reason for adding this noop? It doesn't seem to be used, or mentioned in anywhere.

Reply:
It's not a no-op, it's the flag being added; see the above one for an another example of such flag: https://github.com/nodejs/node/blob/3860002ac678370756228eb0b96e10a2072a4494/src/node_options.cc#L389-L393

That being said, it looks like I forgot to switch the default value to true